alloy-core/src/test/java/org/starchartlabs/alloy/test/core/PreconditionsTest.java

```diff
@@ -83,6 +112,35 @@ public void checkArgumentFalseSupplier() throws Exception {
         Preconditions.checkArgument(false, () -> "message");
     }
 
```

Comment:
We should add a test for the Supplier itself being null (`Preconditions.checkArgument("value", t -> true, (Supplier<String>) null)`)

Reply:
Test added